package.json

```diff
@@ -42,8 +42,10 @@
     "extends browserslist-config-monday"
   ],
   "dependencies": {
+    "@types/react": "^18.2.28",
     "browserslist-config-monday": "^1.0.6",
-    "classnames": "^2.3.2"
+    "classnames": "^2.3.2",
+    "vibe-storybook-components": "^0.9.0"
```

Comment:
Looks like mistake - it's the same package
Please, revert

Reply:
Uninstalled vibe-storybook-components